content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,410 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "Deploy an HA Cluster"
+weight: "1005"
+---
+
+In this guide, we'll walk through the steps neede to take enable Kubernete's high availability capabilities with a cluster installed by Kurl and managed by Kots. This only applies to 'embedded' cluster installs using Kurl.
```

Comment:
Fixing some typos and splitting to two lines, we prefer "one line per sentence" (I admit we don't follow this rule well though)
```suggestion
In this guide, we'll walk through the steps needed to take enable Kubernetes's high availability capabilities with a cluster installed by kURL and managed by KOTS. 
This only applies to 'embedded' cluster installs using Kurl.
```